consoleApp/src/main/java/com/infoshare/service/VolunteerRepository.java

```diff
@@ -0,0 +1,74 @@
+package com.infoshare.service;
+
+import com.infoshare.domain.TypeOfHelp;
+import com.infoshare.domain.Volunteer;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class VolunteerRepository {
+
+    private static final String DB_FILE = "Volunteer.csv";
+
+    public void printFilteredList(){
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Z jakiego miasta chcesz wyszukac wolontariuszy?");
```

Comment:
Wartosc stringa jest dobrym kandydatem do stałej.

Reply:
Nie rozumiem, po co robić to jako stałą i jak miałoby to wyglądać?

Reply:
np na poziomie metody jako : 
final String CITY_CHOSE_HEADER = ""Z jakiego miasta chcesz wyszukac wolontariuszy?"; albo na poziomie klasy. Zwieksza to czytelnosc kodu i latwiej sie go utrzymuje.